metricbeat/values.yaml

```diff
@@ -7,19 +7,36 @@ metricbeatConfig:
       hostfs: /hostfs
     metricbeat.modules:
     - module: kubernetes
+      labels.dedot: true
+      annotations.dedot: true
+      add_metadata: true
```

Comment:
any specific reason to replace `add_kubernetes_metadata` processor by module kubernetes metadata?

Reply:
This PR come from following the given directions and not getting any metrics. In the process of trying to get it working, I noticed that the config in the helmchart didn't match https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-kubernetes.html and changing it was one of a number of things I did to get it working. I know that the `dedot` stuff was necessary. It is possible that I just misunderstood something though- I assumed the helm config simply hadn't kept up to date with the metricbeat settings... but if this isn't right, happy to change it back :)